### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 4

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#317] [Esercizi - Viaggi/Piazza]
![Screenshot 2023-11-12 150813](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/02dc49c3-599c-4d10-ba15-c43db7c1a629)

**Comment by CasuFrost** [comment]
Se ho fatto qualche errore perfavore fatemelo notare.
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/97389762/6da38f31-314b-48ff-8f02-13f4a05d7f12)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/5fadf113-27c2-411a-9424-239421cbb4b0)
